config/env_spec_request.go

```diff
@@ -30,6 +30,11 @@ import (
 
 const TruncateDebugRequestValuesAt = 5
 
+// a "match all" operation for apis without operations
+var synthesizedOperation = &APIOperation{
+	Name: "*",
```

Comment:
Just wanted to confirm, do we indeed disallow such a name?

Reply:
nvm, I realize it doesn't matter.

Reply:
We don't validate against it, but since it's only used when there are no operations, it can't conflict with anything.